server/GraknServer.java

```diff
@@ -127,6 +130,16 @@ private Path dataDir() {
         return command.dataDir();
     }
 
+    private void configureAndVerifyJavaVersion() {
+        int javaMajorVersion = Java.getMajorVersion();
+        if (javaMajorVersion == Java.UNKNOWN_VERSION) {
+            LOG.warn("Could not detect Java version from version string '{}'", System.getProperty("java.version"));
```

Comment:
`Could not detect Java version from version string '{}'. Will start Grakn Core anyway.`

Reply:
@lolski addressed in https://github.com/graknlabs/grakn/pull/6251/commits/e43116bd6946d31ea988551711a5435d37906db6